### ISSUE TITLE
Parser doesn't handle empty strings correctly

### ISSUE THREAD

**Issue by sapphyrus** [node-steam/vdf#1]
Parsing the following VDF File:
```vdf
"test"
{
	"empty_key" ""
}
```
results in this:
```json
{
    "test": {
        "empty_key": 0
    }
}
```

This is because `isNaN`, for some reason, returns false for empty strings

![](https://i.imgur.com/psaKlPM.png)

https://github.com/node-steam/vdf/blob/fad53d044f01e30bfb1b383405a77eb45d6b07f5/src/index.ts#L61

**Comment by lenovouser** [comment]
Thanks!

**Comment by lenovouser** [comment]
Released in [`@node-steam/vdf@2.1.0`](https://yarnpkg.com/en/package/@node-steam/vdf)
